src/Compilers/VisualBasic/Portable/Lowering/UseTwiceRewriter.vb

```diff
@@ -208,7 +208,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Debug.Assert(node.IsLValue)
 
-#If DONT_USE_BYREF_LOCALS_FOR_USE_TWICE Then
```

Comment:
It is not clear why is this conditional compilation got removed from this method. I would expect it to be adjusted to ensure that UseTwiceLValue is still not called when DONT_USE_BYREF_LOCALS_FOR_USE_TWICE is defined.

Reply:
Added `#If`.